app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
> I'm skeptical of putting view logic in the model.

Me too, my wording was poor there, I was not intending to talk about the track model. It would be some kind of independently testable "topic view/decorator" object.